deployment/ansible/group_vars/production.j2

```diff
@@ -21,6 +21,11 @@ app_version: {{ app_version }}
 # instances are configured to use https.
 app_domain_name: {{ app_domain_name }}
 
+## DOCKER SETTINGS
+# The docker repository you want your app to pull images from. The default is "quay.io/azavea/".
+# The trailing / is required.
+docker_repository: {{ docker_repository }}
```

Comment:
Hey, I chatted with @pcaisse about this and I think we can just have this line be `docker_repository: "quay.io/azavea/"` -- the wizard doesn't need to prompt for it. There are a bunch of things that aren't covered by the wizard that people will need to go into the group_vars to edit manually anyway, so if they ever need to fork the app they'll probably have a pretty good command of the group_vars by that point.